src/Quartz/Impl/Triggers/CalendarIntervalTriggerImpl.cs

```diff
@@ -62,6 +62,7 @@ public sealed class CalendarIntervalTriggerImpl : AbstractTrigger, ICalendarInte
     private DateTimeOffset? previousFireTimeUtc;
     private int repeatInterval;
     internal TimeZoneInfo? timeZone;
+    private const int TimeZoneAdjustmentIntervalMinutes = 15;  //timezones can be multiples of 15 minutes (30min, 45min, 1hr)
```

Comment:
I wonder instead of this interval, if we just find the DST difference and add that amount, so if previous DST to new DST is +1hr , add 60 minutes ?